vividus-extension-selenium/src/main/java/org/vividus/selenium/ControllingMetaTag.java

```diff
@@ -104,4 +104,9 @@ public static void setDesiredCapabilitiesFromMeta(DesiredCapabilities capabiliti
             controllingMetaTag.setCapability(capabilities, meta);
         }
     }
+
+    public static boolean isContainedIn(Meta meta, String metaName)
```

Comment:
no need to make this method a part of ControllingMetaTag

Reply:
moved